common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -183,6 +184,7 @@ class CapaFields(object):
                        scope=Scope.user_state, default={})
     input_state = Dict(help=_("Dictionary for maintaining the state of inputtypes"), scope=Scope.user_state)
     student_answers = Dict(help=_("Dictionary with the current student responses"), scope=Scope.user_state)
+    score = ScoreField(help=_("Dictionary with the current student score"), scope=Scope.user_state, enforce_type=False)
```

Comment:
Please add a comment about why it's okay to have `enforce_type=False` here, since it's not often used.